modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -27,13 +27,14 @@ Create assembly and module file names that accurately and closely reflect the ti
 Do not include special characters in file names. Ensure that all members of your team use the same file naming conventions.
 ====
 
-These file naming guidelines are optional but highly recommended. However, if your team does not include the module prefixes in file names followed by either a hyphen (-) or an underscore (_), include one of the following variables in each concept, procedure, and reference module before the module ID:
+These file naming guidelines are optional but highly recommended. However, if your team does not include the module prefixes in file names followed by either a hyphen (-) or an underscore (_), include one of the following variables in each assembly file and concept, procedure, and reference module before the module ID:
 
 [source]
 ----
-:_module-type: CONCEPT
-:_module-type: PROCEDURE
-:_module-type: REFERENCE
+:_content-type: ASSEMBLY
+:_content-type: CONCEPT
+:_content-type: PROCEDURE
+:_content-type: REFERENCE
```

Comment:
Just a side note that I think is outside of the scope of this PR. We might want to update this section to also mention snippets, and include a snippet file in the example file names previously in this section, and then add a content-type of SNIPPET to this list. But snippets are still fairly new territory, so not sure if we're to a point that we want to mention it beyond the snippets section later in this guide, which you've correctly updated. So I think good for now unless someone wants to expand this section to include snippets in a separate PR.